sunpy/physics/limb_darkening.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+
+def darklimb_u(wavelength):
+    """
+    Returns the constant, u, for a given wavelength for input into the limb
+    darkening function.
+
+    Parameters
+    ----------
+    wavelength : `~sunpy.map.GenericMap.wavelength`
+        The wavelength for which the limb darkening is to be calculated.
+
+    Notes
+    -----
+    Coefficients for 5th order polynomial fit to limb darkening
+    constant `u` was done on data from Astrophysical
+    Quantities by Allen.
+
+    Limitations
+    -----------
+    Only applies to wavelengths in the range 4000<lambda<15000 Angstrom
```

Comment:
I am not sure if theses should be sections and not just part of the first paragraph.